packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -4,7 +4,7 @@
 
 import 'dart:async';
 
-import 'package:meta/meta.dart' show required;
+import 'package:meta/meta.dart' show required, visibleForTesting;
```

Comment:
you can also just remove the `show` line, we usually import this file unconditionally.